src/libraries/System.Security.Cryptography.ProtectedData/src/System/Security/Cryptography/ProtectedData.cs

```diff
@@ -29,23 +173,174 @@ public static byte[] Unprotect(byte[] encryptedData, byte[]? optionalEntropy, Da
             if (encryptedData is null)
                 throw new ArgumentNullException(nameof(encryptedData));
 
-            return ProtectOrUnprotect(encryptedData, optionalEntropy, scope, protect: false);
+            byte[]? outputData;
+            bool result = TryProtectOrUnprotect(
+                encryptedData,
+                optionalEntropy,
+                scope,
+                protect: false,
+                allocateArray: true,
+                bytesWritten: out _,
+                outputData: out outputData
+            );
+
+            Debug.Assert(result);
+            Debug.Assert(outputData != null);
+            return outputData;
+        }
+
+        /// <summary>
+        /// Decrypts the data in a specified byte array and returns a byte array that contains the decrypted.
```

Comment:
```suggestion
        /// Decrypts the data in a specified byte array and returns a byte array that contains the decrypted data.
```

Looks like the last word got lost